### PR TITLE
[IMP] product_barcodelookup: product attribute improvements

### PR DESCRIPTION
**Pull request by sjai-odoo**
- We have given visibility to Age Group and Manufecturer attributes
created in product_barcodelookup.

Related: https://github.com/odoo/enterprise/pull/63740/
task- 3965079

### CODE REVIEW

**Comment by Feyensv** [comment]
> * We have created some of the necessary attributes, enabling their automatic creation when they are received from barcode lookup.

What do you mean by necessary ?
The new records you are creating are only in demo data files, so they probably aren't necessary :thinking:.
Also, if they are used by the barcodelookup module, why don't you specify it in the module directly?

**Comment by sjai-odoo** [comment]
> > * We have created some of the necessary attributes, enabling their automatic creation when they are received from barcode lookup.
> 
> What do you mean by necessary ? The new records you are creating are only in demo data files, so they probably aren't necessary 🤔. Also, if they are used by the barcodelookup module, why don't you specify it in the module directly?

Hi @Feyensv ,

I'm sorry for the mistake. I first put the data in the demo files by accident, but now I've moved it to the data file of the product_barcodelookup module. I've also updated the commit message to make it clearer.

Thanks.

File: addons/website_sale/data/data.xml [hunk 1]
```diff
@@ -22,6 +22,12 @@
             <field name="name">Brand</field>
             <field name="sequence">0</field>
         </record>
+        <record id="product_barcodelookup.product_attribute_5" model="product.attribute">
+            <field name="visibility">hidden</field>
+        </record>
+        <record id="product_barcodelookup.product_attribute_6" model="product.attribute">
+            <field name="visibility">hidden</field>
+        </record>
```
Comment (Feyensv) [inline addons/website_sale/data/data.xml:30]: Sadly, you cannot reference a record which is not defined in the module dependencies (does it really matter to hide those though) ?

Comment (sjai-odoo) [reply addons/website_sale/data/data.xml]: I have changed the code accordingly now we will create these attributes when encountered but not available.

**Comment by sjai-odoo** [comment]
Handled this in [enterprise](https://github.com/odoo/enterprise/pull/63740), so closing this PR.

**Pull request closed without merging**